dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -324,6 +324,14 @@ def python_versions(self) -> list[str]:
     def python_versions_list_as_string(self) -> str:
         return " ".join(self.python_versions)
 
+    @cached_property
+    def min_max_python_version_as_string(self) -> str:
```

Comment:
Looking closer, naming wise, not sure this matches the behavior. Maybe `basic_python_versions_as_string`? `min_max_or_default_python_versions_as_string`?

Reply:
It actually this  "or default" is optimization for regular build where min = max - so the logic is a little deceptive, but the meaning is actually what it is in the name.

Reply:
We hava very similar approach in "current_*_version" - when full build is run - this is the whole range, but when "simple" PR -> this is jus the default

Reply:
But I just noticed I missed "s" at the end :)